manual/graphs/index.md

```diff
@@ -11,7 +11,7 @@ A graph is an object made of two arrays `X` and `Y`, holding the `x`,`y` coordin
 **Un-binned data**
 
 A graph or chart is a set of categorical variables, this is un-binned data.<br>
-A histogram is used for continuous data, where the bins represent ranges of data (binned data), see → [Histograms]({{ '/manual/histograms' | relative_url }}).
+A histogram is used for continuous data, where the bins represent ranges of data (binned data), see → [Fitting histograms](/manual/fitting).
```

Comment:
we probably don't use them consistently enough, but links generated with `{{ .. | relative_url }}` are the recommended way, see https://jekyllrb.com/docs/github-pages/#project-page-url-structure

Reply:
Thanks, good to know! But do we care about this? `jekyll serve` does the right thing, the linkchecker understand the simpler way to put links and also the deployed website is happy.

Reply:
i don't know what will break without the relative_url, but if they say that's the right way, why should we go out of our way to do it differently? :D

as an example, here is an account of things went wrong without relative_url: https://github.com/github/pages-gem/issues/460

Reply:
But that's only for the baseurl `"/"`. But fair enough, let me make an issue to keep track.

Reply:
https://github.com/root-project/web/issues/208